jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ContainerizerIntegrationTest.java

```diff
@@ -303,7 +303,9 @@ private JibContainer buildRegistryImage(
       throws IOException, InterruptedException, RegistryException, CacheDirectoryCreationException,
           ExecutionException {
     return buildImage(
-        baseImage, Containerizer.to(RegistryImage.named(targetImage)), additionalTags);
+        baseImage,
+        Containerizer.to(RegistryImage.named(targetImage)).setAlwaysCacheBaseImage(true),
```

Comment:
I see you don't do anything about after setting this. If so, I think there's no reason to change the current test behavior. This is important because I believe there are tests for `jib.alwaysCacheBaseImage` somewhere and this could nullify the very existence of those tests.

Reply:
Agree and removed.